src/RequestHeaders.php

```diff
@@ -24,11 +24,13 @@ public function getAll(): array
     /**
      * Get values for a specific header with a specific key.
      * @param string $key
-     * @return array<string>
+     * @return array<string>|null
      */
-    public function get(string $key): array
+    public function get(string $key): ?array
```

Comment:
Previously returned empty array for header keys that don't exist. Thought that would create the impression that the header exists but is empty. Introduced returning `null` instead. Thoughts?

Reply:
Or throw an exception?